app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1030,8 +1032,11 @@ protected void deliverError(CommCareHomeActivity receiver, Exception e) {
     }
 
     @Override
-    protected void onResume() {
-        super.onResume();
+    protected void onResumeFragments() {
```

Comment:
Supposedly `FragmentActivity` should override `onResumeFragments` not `onResume` whenever possible.